ansible/roles/common/templates/ssh/sshd_config

```diff
@@ -34,7 +34,7 @@ AcceptEnv LANG LC_*
 
 Subsystem sftp /usr/lib/openssh/sftp-server
 
-AllowGroups allow-ssh
+AllowGroups allow-ssh dev-desktop-allow-ssh
```

Comment:
Can we only add this on the dev-desktop machines rather than all machines? Maybe adding an `extra-ssh-allow-groups` variable.

Reply:
I don't know how to do that.

Appending sshd_config with files in sshd_config.d has been very weird with overwriting previous flags. I'll try it out, but if that doesn't work, we should imo just allow this group or rename it to something generic like `allow-ssh-noninfra`

Reply:
I would add this to the [default variables for the Ansible role](https://github.com/rust-lang/simpleinfra/blob/master/ansible/roles/common/defaults/main.yml):

```yaml
allow_ssh_extra_groups: ""
```

Then the current line in the `sshd_config` can be replaced with:

```
AllowGroups allow-ssh {{ allow_ssh_extra_groups }}
```

And finally in the playbook, in the common role we add:

```yaml
- role: common
  # ...
  allow_ssh_extra_groups: dev-desktop-allow-ssh
```

I agree it's not worth it messing with `sshd_config.d`.

Reply:
Oooh, these templates are great. Thanks for the explanation